ghost/core/test/utils/fixtures/fixtures.json

```diff
@@ -1123,6 +1123,28 @@
                 "sell": ["Getting Started"]
             }
         },
+        {
+            "from": {
+                "model": "Collection",
+                "match": "slug",
+                "relation": "posts"
+            },
+            "to": {
+                "model": "Post",
+                "match": "slug"
+            },
+            "entries": {
+                "latest": {
+                    "integrations": "integrations",
```

Comment:
This notation was pretty counter intuitive to figure out... Basically we have to have a duplicate "key" and "value" here for [matching to work](https://github.com/TryGhost/Ghost/blob/57a851227cad6e30c400636daf3bed517078de75/ghost/core/core/server/data/schema/fixtures/FixtureManager.js#L43). And the `key = key === 0 && value ? value : key;` in the matcher makes the matching logic even more confusing as it kinda works for the first record if passed in as an array, but than stops working when more elements are added.

Reply:
Reading the tests somewhat helps to understand the logic. But initially it was pretty confusing getting this right as the expected intuitive syntax like:
```js
"latest": ["integrations", "grow", "sell", ....]
```
didn't work out of the box.